pylmod/client.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+"""
+Client classes and methods for PyLmod module
+"""
+
+import logging
+
+log = logging.getLogger('pylmod.client')  # pylint: disable=invalid-name
+
+class Client(object):
+
+    def get_academicterms(self):
+        pass
+
+
+    def get_assignment_by_name(self):
+        pass
+
+
+    def get_assignments(self):
+        pass
+
+
+    def get_gradebookid(self):
+        pass
+
+
+    def get_grades(self):
+        pass
+
+
+    def get_section_by_name(self):
+        pass
+
+
+    def get_sections(self):
+        pass
+
+
+    def get_student_by_email(self):
+        pass
+
+
+    def get_students(self):
+        pass
+
+
+    def create_assignment(self):
+        pass
+
+
+    def delete_assignment(self):
+        pass
+
+
```

Comment:
If we are doing a rough draft of the API, should there be things like set_score and such here?